Adding some logging to enhance metrics capabilities

mcarrier7 opened the pull request:
Adding some logging to enhance metrics capabilities, specifically logging for job execution times and health checks

go0ty commented on src/main/java/org/venice/beachfront/bfapi/services/JobPoller.java:
```diff
@@ -191,34 +192,38 @@ public void processJobStatus(Job job, StatusMetadata status) {
+					// mark the log as a success and how long it took to complete
 					piazzaLogger.log(
-							String.format("Successfully recorded Detection geometry for Job %s and marking as Success.", job.getJobId()),
+							String.format("Successfully recorded Detection geometry for Job %s and marking as Success. Job completed in %d ms", job.getJobId(), 
```
Can we use some Joda time here? I feel like milliseconds is not important here.

Something like: 

new Duration(createdOn, now).getStandardMinutes()

Minutes will make for much more readable logs , I feel like.

go0ty replied:
You can put this code in one place at the top, before the switch on statuses.

mcarrier7 commented:
I moved the job completion message to the top, and changed the diff time unit to minutes, like suggested. I was trying to differentiate between jobs which succeeded and failed due to errors, though I added some logic to the log message at the top to achieve this.
